randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -140,11 +152,14 @@ Extra - asset_id: 3
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
       Any of the following:
-          # Via top path - TODO: check for shinespark without entrance rando?
-          Grapple Beam or Space Jump
+          Any of the following:
+              # Via top path
+              Grapple Beam or Space Jump or Movement (Intermediate)
+              # 2 tap short charge from West Ocean
+              Speed Booster and Shinespark (Beginner) and Short Charge (Intermediate) and Shinespark Damage ≥ 43 and Disabled Entrance Randomizer
```

Comment:
Is Shinesparking necessary? Wouldnt you be able to go over with just jump and the momentum from speed?

Reply:
You're right, removed the spark.